ibis/backends/dask/__init__.py

```diff
@@ -12,8 +12,9 @@
 
 from .client import DaskClient, DaskTable
 from .execution import execute, execute_node
+from .udf import udf
 
-__all__ = ('connect', 'dialect', 'execute')
+__all__ = ('connect', 'dialect', 'execute', 'udf')
```

Comment:
is this similar for spark/pandas (e.g. where its exposed)?

Reply:
Matches what the pandas backend exposes: https://github.com/ibis-project/ibis/blob/master/ibis/backends/pandas/__init__.py#L13

Reply:
Can you use the API under https://github.com/ibis-project/ibis/blob/master/ibis/udf/vectorized.py#L102?

Reply:
This just creates an alias to that API: https://github.com/ibis-project/ibis/blob/master/ibis/backends/pandas/udf.py#L137. 

Happy to do anything - was just working off the pandas API as a template.

Reply:
I am leaning towards using the API under udf.vectorized, it's more standard  since these UDFs works cross backends.

Reply:
I actually feel that we should keep the pandas and dask interfaces the same.

Reply:
IMHO having the code between Spark, Pandas, Dask the same is better than keeping Pandas and Dask the same but different from Spark.

The reason that pandas still have backends.pandas.udf is purely for backward compatible reasons. We should really deprecate it in favor of the vectorized.udf interface.

Reply:
ok i agree at the end of the day we want the same interface for all backends. so if `udf.vectorized` is is, then let's do that (@gerrymanoim if you wouldn't mind creating an issue for deprecating the old pandas udf interface)